src/css/global/layout.scss

```diff
@@ -0,0 +1,11 @@
+// Override Protocol's 25% / 75% layout to be 33% / 67%
+
+@media #{$mq-md} {
+  .mzp-l-sidebar {
+    width: 33.333% !important;
```

Comment:
Is this to style this container? https://github.com/mdn/developer-portal/pull/1285/files#diff-b7ede9403e1fba8bb7a9047a07b700e1R5

If so, could we do:

```
aside.mzp-l-sidebar {
  width: 33.333%;
}
```

Reply:
We'll be using that override in other templates too, but it will always be an aside, so I could make it more specific, sure.

(Or are you saying it doesn't need to be in a media query and we just override it everywhere? I didn't do that because I want it to flow as above-and-below in small viewports)